src/traditional_de/estimate_delay.jl

```diff
@@ -119,11 +119,16 @@ end
 #####################################################################################
 #                               Mutual information                                  #
 #####################################################################################
-export mutualinformation
+export selfmutualinfo, mutualinformation
+function mutualinformation(args...; kwargs...)
+    @warn "`mutualinformation` is deprecated in favor of `selfmutualinfo`."
```

Comment:
This might go into a different file: `deprecate.jl`. I like having deprecated functions methods in a different place, because it helps other people to have a clear perspective of the situation.

Reply:
sure!